### ISSUE TITLE
Using --debug with other options does not work

### ISSUE THREAD

**Issue by turnstep** [keithf4/pg_bloat_check#27]
I can use `--debug` just fine if the only option.

I can use `--table pgbench_accounts` fine as the only option.

Combined, they give an error:
```

$ pg_bloat_check.py --table pgbench_accounts --debug
quiet level: 0
Traceback (most recent call last):
  File "/bin/pg_bloat_check.py", line 698, in <module>
    get_bloat(conn, tuple(exclude_schema_list), tuple(include_schema_list), exclude_object_list)
  File "/bin/pg_bloat_check.py", line 201, in get_bloat
    print("sql_class: " + cur.mogrify(sql_class, [args.tablename, args.tablename] ) )
TypeError: must be str, not bytes

```
Version is 2.7.0

**Comment by keithf4** [comment]
Pushed a fix to branch https://github.com/keithf4/pg_bloat_check/tree/2.7.1

Found a few other spots that were missing the conversion as well. Let me know if that works ok now.

**Comment by turnstep** [comment]
Working now, thanks!

**Comment by keithf4** [comment]
Released version 2.7.1 with fix. Thanks for reporting and testing!
